lib/desktop/session.rb

```diff
@@ -288,6 +299,7 @@ def start_vnc_server(geometry: Config.geometry, postinitscript: nil)
       if postinitscript
         args << '-postinitscript' << postinitscript
       end
+      env_override = override_env || !no_override_env && Config.session_env_override
```

Comment:
Would this not be better located `commands/start.rb`? That way only a single authoritative option would be need to be passed through the various `session.rb` calls.